server/controllers/bookController.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable require-jsdoc */
+import models from '../db/models';
+import Utils from '../helpers/utilities';
+
+
+class BookController {
+  static async addBook(req, res) {
+    const {
+      title,
+      description,
+      amount,
+      authorID,
+      status,
+      year
+    } = req.body;
+    try {
+      const book = await models.Books.create({
+        title,
+        description,
+        amount,
+        authorID,
+        status,
+        year,
+      });
+
+      return Utils.successStatus(res, 201, 'Book added successfully', {
```

Comment:
Return the book object directly here. There's no need of calling out each property since you're sending all of them back in the response